devito/passes/clusters/misc.py

```diff
@@ -1,10 +1,11 @@
+from collections import Counter
```

Comment:
can we come up with  a better name than `misc` like manipulation

Reply:
we could, but all files in this subdirectory contain "manipulation" routines, so it's a bit odd?

Reply:
True, just find misc bit too vague, food for thought